### ISSUE TITLE
Getting NullPointerException exception periodically

### ISSUE THREAD

**Issue by hrstoyanov** [orientechnologies/orientdb#8375]
### OS: MacOS,Windows

After leaving the console connected to my server for a while (like overnight), I get:
```
Exception in thread "Thread-4" java.lang.NullPointerException
        at com.orientechnologies.orient.client.remote.OStorageRemotePushThread.subscribe(OStorageRemotePushThread.java:109)
        at com.orientechnologies.orient.client.remote.OStorageRemote.subscribeDistributedConfiguration(OStorageRemote.java:1450)
        at com.orientechnologies.orient.client.remote.OStorageRemote.onPushReconnect(OStorageRemote.java:2072)
        at com.orientechnologies.orient.client.remote.OStorageRemotePushThread.run(OStorageRemotePushThread.java:86)

```

**Comment by avitaleOPEN** [comment]
I'm having the same behaviour. After a couple of hours working with the database (an actors system which writes and reads the database) I get the exception.

**Comment by tglman** [comment]
Hi @hrstoyanov,

I just fixed this in 3.0.5-SNAPSHOT.

Regards
